aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ConnectEvent.java

```diff
@@ -90,12 +90,37 @@ public static class ContactData implements Serializable {
         private String previousContactId;
 
         @JsonProperty("Queue")
-        private String queue;
+        private Queue queue;
 
         @JsonProperty("SystemEndpoint")
         private SystemEndpoint systemEndpoint;
     }
 
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class Queue implements Serializable, Cloneable {
```

Comment:
For EventsV4 we decided to drop Cloneable interface, can you please remove it?

Reply:
Removed